Assets/Scripts/BulletManagement/Patterns/OctoStraight.cs

```diff
@@ -1,21 +1,22 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
+﻿using UnityEngine;
 
-/**
- * Emitts a pattern from the center in a straight line with 8 spokes.
- */
-public class OctoStraight : BulletPattern
+namespace BulletManagement.Patterns
 {
-    public override Vector3 GetTranslation(float time, int bulletId)
+    /**
+    * Emits a pattern from the center in a straight line with 8 spokes.
+    */
+    public class OctoStraight : BulletPattern
     {
-        float spokeCount = bulletId % 8;
+        public override Vector3 GetTranslation(float time, int bulletId)
+        {
+            float spokeCount = bulletId % 8;
 
-        float c = (Mathf.PI / 4) * spokeCount;
-        // Derivative functions with respect to time
-        float dx = Mathf.Cos(c);
-        float dy = Mathf.Sin(c);
+            var c = Mathf.PI / 4 * spokeCount;
```

Comment:
Same here, what is the reasoning behind `var`?

Reply:
See above.

Reply:
It seems to me it is not easily understood what `Mathf` returns in this case. I think I would prefer to leave this in so we know the precision of the numbers returned.

Reply:
Precision is pretty important here since we want to work with translation.

Reply:
Right, but the compiler will complain at you if you try to change the type. The compiler is the one that's telling me that the type specification isn't needed. This is how C# works.

Reply:
Changed the warning setting on my IDE to prefer explicit typing in some cases, which addressed this case.